### ISSUE TITLE
4.2 comparability question

### ISSUE THREAD

**Issue by ecottis** [catalyst/moodle-mod_observation#126]
Is this plugin available or will it me made available for Moodle Version 4.2

**Comment by matthewhilton** [comment]
Yes, use the https://github.com/catalyst/moodle-mod_observation/tree/MOODLE_400_STABLE branch which is compatible with 4.2.
